code/game/machinery/camera/camera.dm

```diff
@@ -14,17 +14,19 @@
 	max_integrity = 100
 	integrity_failure = 50
 	var/list/network = list("SS13")
+	var/list/previous_network
 	var/c_tag = null
 	var/c_tag_order = 999
 	var/status = TRUE
 	anchored = TRUE
 	var/start_active = FALSE //If it ignores the random chance to start broken on round start
 	var/invuln = null
 	var/obj/item/camera_assembly/assembly = null
+	var/part_of_camera_network
```

Comment:
Can you autodoc this please? It's not immediately obvious what this var is or holds

Reply:
Should be a bit clearer I hope